src/unlimited.h

```diff
@@ -196,6 +201,7 @@ extern CTweak<uint64_t> blockSigopsPerMb;
 extern CTweak<uint64_t> coinbaseReserve;
 extern CTweak<uint64_t> blockMiningSigopsPerMb;
 
+extern std::list<CStatBase*> mallocedStats;
```

Comment:
I hear you but I don't think that these issues are important enough for me spend the time to switch from list to vector.  If you'd like to switch it, go ahead!  You can check out my branch in your repo, and then issue a PR into my branch.  Or after this gets merged, do it the separately.